src/rabbit_mirror_queue_misc.erl

```diff
@@ -39,10 +39,15 @@
      {mfa, {rabbit_registry, register,
             [policy_validator, <<"ha-sync-mode">>, ?MODULE]}},
      {mfa, {rabbit_registry, register,
+            [policy_validator, <<"ha-sync-batch-size">>, ?MODULE]}},
+     {mfa, {rabbit_registry, register,
             [policy_validator, <<"ha-promote-on-shutdown">>, ?MODULE]}},
      {requires, rabbit_registry},
      {enables, recovery}]}).
 
+%% For compatibility with versions that don't support sync batching.
+-define(DEFAULT_BATCH_SIZE, 1).
```

Comment:
You are right. I think this constant came to life on my first POCs, but is not required anymore. Probably not used in the code.

> On Oct 10, 2015, at 3:28 PM, Michael Klishin <notifications@github.com> wrote:
> 
> In src/rabbit_mirror_queue_misc.erl:
> 
> >              [policy_validator, <<"ha-promote-on-shutdown">>, ?MODULE]}},
> >       {requires, rabbit_registry},
> >       {enables, recovery}]}).
> >  
> > +%% For compatibility with versions that don't support sync batching.
> > +-define(DEFAULT_BATCH_SIZE, 1).
> Are we talking about pre-3.6.0 versions here? Mixed 3.6.0/3.5.x clusters are not allowed, so we can use a different default.
> 
> —
> Reply to this email directly or view it on GitHub.
>